tests/tpu/test_moe_pallas.py

```diff
@@ -26,7 +26,7 @@
 # The Pallas GMM kernel requires num_tokens * topk to be a multiple of 16
 @pytest.mark.parametrize("m", [8, 16, 64, 2048])
 @pytest.mark.parametrize("n", [128, 1024, 2048])
-@pytest.mark.parametrize("k", [128, 511, 1024])
+@pytest.mark.parametrize("k", [128, 512, 1024])
```

Comment:
Is there a divisibility requirement on all of these? It would be good to test some cases where they aren't required to be easy powers of 2